### PR TITLE
OSDOCS-1837: Remove admonition that TLS 1.3 and  config are not supported in Ingress

### PR DESCRIPTION
**Pull request by skrthomas**
https://issues.redhat.com/browse/NE-472

TLS 1.3 is now supported in Ingress, so I have removed the admonition stating that it is not supported.

For 4.9

Preview: https://deploy-preview-35361--osdocs.netlify.app/openshift-enterprise/latest/networking/ingress-operator?utm_source=github&utm_campaign=bot_dp#configuring-ingress-controller-tls

@miheer

### CODE REVIEW

File: modules/tls-profiles-understanding.adoc [hunk 1]
```diff
@@ -35,11 +35,6 @@ The `Intermediate` profile requires a minimum TLS version of 1.2.
 
 The `Modern` profile requires a minimum TLS version of 1.3.
 
-[IMPORTANT]
```
Comment (skrthomas) [inline modules/tls-profiles-understanding.adoc:38]: This information is not in the Networking area, but I found it when searching the documentation for the `Modern` parameter. I'm unsure if removing here is too far-reaching?

**Comment by miheer** [comment]
@skrthomas LGTM

**Review by codyhoag — APPROVED** [review]
LGTM!

**Comment by miheer** [comment]
@skrthomas In OCP 4.6, 4.7 and 4.8,

The “Modern” profile will continue to be unsupported on OpenShift 4.6, 4.7, and 4.8; selecting it will continue to have the effect of enabling the “Intermediate” profile.
Specifying TLSv1.3 cipher suites in Custom Profile will continue to have no effect; OpenShift router will enable Red Hat distributed OpenSSL’s default set of TLSv1.3 cipher suites which are TLS_AES_128_CCM_SHA256, TLS_CHACHA20_POLY1305_SHA256, TLS_AES_256_GCM_SHA384, TLS_AES_128_GCM_SHA256
Users need to aware that their clusters may be accepting TLSv1.3 connections and cipher suites when they did not expect this behavior.

**Comment by skrthomas** [comment]
> @skrthomas In OCP 4.6, 4.7 and 4.8,
> 
> The “Modern” profile will continue to be unsupported on OpenShift 4.6, 4.7, and 4.8; selecting it will continue to have the effect of enabling the “Intermediate” profile. Specifying TLSv1.3 cipher suites in Custom Profile will continue to have no effect; OpenShift router will enable Red Hat distributed OpenSSL’s default set of TLSv1.3 cipher suites which are TLS_AES_128_CCM_SHA256, TLS_CHACHA20_POLY1305_SHA256, TLS_AES_256_GCM_SHA384, TLS_AES_128_GCM_SHA256 Users need to aware that their clusters may be accepting TLSv1.3 connections and cipher suites when they did not expect this behavior.

I'm addressing this comment in a separate PR that is for 4.6-4.8.

File: modules/nw-ingress-controller-configuration-parameters.adoc [hunk 2]
```diff
@@ -92,18 +92,18 @@ If not set, the default value is based on the `apiservers.config.openshift.io/cl
...
-
-The Ingress Operator also converts the TLS `1.0` of an `Old` or `Custom` profile to `1.1`, and TLS `1.3` of a `Custom` profile to `1.2`.
+Ciphers and the minimum TLS version of the configured security profile are reflected in the `TLSProfile` status.
 ====
 
-[NOTE]
+[IMPORTANT]
 ====
-Ciphers and the minimum TLS version of the configured security profile are reflected in the `TLSProfile` status.
+The `Modern` profile requires TLS `1.3`.
```
Comment (skrthomas) [inline modules/nw-ingress-controller-configuration-parameters.adoc:104]: @miheer this is the text I'm referencing in our Slack message

**Pull request merged**